GlobalPlanner: 
  verbose: false
  plan_with_footprint: false
  plan_with_constant_heading: false
  heading: 0.0
  allow_backward_motion: false

  Dijkstra: 
    use_dijkstra: true
    allow_unknown: false
    default_tolerance: 0.0
    old_navfn_behavior: false
    use_quadratic: true
    use_grid_path: false
    visualize_potential: false

  RRTConnect:
    footprint_topic: "/squirrel_footprint_observer/footprint"
    bold_factor: 0.05
    collision_check_resolution: 0.01
    max_planning_time: 0.2
    max_simplification_time: 0.1
    map_resolution: 0.05
    range: 1.0
    verbose: false